app/views/pensions/update/WhatDoYouWantToTellUsView.scala.html

```diff
@@ -14,44 +14,61 @@
  * limitations under the License.
  *@
 
-@import includes.forms.{submit, textArea}
-@import includes.{cancelLink, ptaHeader}
+@import includes.forms.submit
+@import includes.cancelLink
 @import uk.gov.hmrc.tai.viewModels.employments.EmploymentViewModel
 @import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 @import uk.gov.hmrc.tai.forms.employments.UpdateEmploymentDetailsForm
 @import uk.gov.hmrc.tai.forms.pensions.WhatDoYouWantToTellUsForm
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukCharacterCount
+@import uk.gov.hmrc.govukfrontend.views.html.components.Text
+@import uk.gov.hmrc.govukfrontend.views.html.components.Hint
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukErrorSummary
+@import uk.gov.hmrc.govukfrontend.views.html.components.ErrorSummary
+@import uk.gov.hmrc.govukfrontend.views.html.components.CharacterCount
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLabel
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcSectionCaption
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
 
-@(name: String, id:Int, whatDoYouWantToTellUsForm: Form[String])(implicit request: Request[_], user: controllers.auth.AuthedUser, messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF, govukCharacterCount : GovukCharacterCount, govukErrorSummary: GovukErrorSummary)
 
+@(name: String, id:Int, whatDoYouWantToTellUsForm: Form[String])(implicit request: Request[_], user: controllers.auth.AuthedUser, messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
-@header = @{
-    ptaHeader(
-        formForErrorSummary = Some(whatDoYouWantToTellUsForm),
-        displayBackLink = true,
-        mainHeadingText = Messages("tai.updatePension.whatDoYouWantToTellUs.heading", name),
-        preHeadingText = Messages("tai.updatePension.preHeading")
-    )
-}
 
 @mainTemplate(
     title = Messages("tai.updatePension.whatDoYouWantToTellUs.pagetitle"),
+    pageTitle = Some(messages("tai.service.navTitle")),
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = None) {
+    backLinkContent = Some(Messages("tai.back-link.upper"))
```

Comment:
if we dont pass in backLinkId it will default to "returnToChooseTaxYearLink", do we want that for this page?

Reply:
my question is why does the default template have that instead of backLink? :D but yes maybe i should change that to backLink (but that means do it in all places u.u)

Reply:
also there is a problem calling it backlink because then there's some JS code that overrides any backLinkId into the JS function

Reply:
hmm shall we say lets make another ticket to change the mainTemplate default ?

Reply:
Going to merge an resolve as part of https://jira.tools.tax.service.gov.uk/browse/PAYEE-533